syncserver/src/server/mod.rs

```diff
@@ -358,6 +358,12 @@ impl Server {
         });
 
         let server = server
+            .worker_max_blocking_threads(settings.actix_threadpool.unwrap_or_else(|| {
```

Comment:
This value differs from `ACTIX_THREADPOOL`'s which set the size of a global pool across the process, whereas this pool is [per actix-web worker](https://docs.rs/actix-web/latest/actix_web/struct.HttpServer.html#method.worker_max_blocking_threads).

So if we want to continue treating this setting as the overall thread count (wfm) this value should be divided by `available_parallelism()` (see [the default](https://github.com/actix/actix-net/blob/8cf79d3d1364cd91fb933640c7614a40586d1eb6/actix-server/src/worker.rs#L256) )